backend/cmd/server.go

```diff
@@ -38,5 +39,6 @@ func main() {
 		oidcClientID:     *oidcClientID,
 		oidcClientSecret: *oidcClientSecret,
 		oidcIdpIssuerURL: *oidcIdpIssuerURL,
+		baseURL:          *baseURL,
```

Comment:
Don't we need to make sure (in this block or another) that the base URL has a starting `/`?

Reply:
This validation has been added.